WORKSPACE

```diff
@@ -86,14 +86,35 @@ load("@io_bazel_rules_python//python:pip.bzl", "pip_import", "pip_repositories")
 
 pip_repositories()
 
-pip_import(
-    name = "pip_dependencies",
-    requirements = "//tools/dependencies:python_requirements.txt",
+#pip_import(
+#    name = "pip_dependencies",
+#    requirements = "//tools/dependencies:python_requirements.txt",
+#)
+#
+## Load the pip_install symbol for my_deps, and create the dependencies repositories.
+#load("@pip_dependencies//:requirements.bzl", "pip_install")
+#
+#pip_install()
+
+## rules_pip
+
+http_archive(
+    name = "com_github_ali5h_rules_pip",
+    strip_prefix = "rules_pip-0.5.0",
```

Comment:
use version 1.0.0

Reply:
Thanks for following up on this, mate. Definitely helped me out. 👏